### PR TITLE
Simplified metadata

### PR DESCRIPTION
**Pull request by DSWallach**
# Pull Request Template for MMEDS

## What has changed
Include each file that was changed, how it was changed (in broad terms), and why it was changed.

`mmeds/config.py` - Modified values for the switch to mmedsadmin
`mmeds/server.py` - Added whitespace striping for all webpage inputs.
                               - Added options for uploading simplified metadata
`mmeds/summary.py` - Added stdout/stderr handling
`mmeds/validate.py` - Fixed an issue where dates were being handled as strings

## Checklist of pre-requisites
-   [x] Does the code run?  
-   [x] Does the code follow the repository style?  
-   [x] Is the code tested?  

## How to use the feature
Example: To do X run new function Y with parameters A, B, and C which gives result Z.

## How the design has changed
Example: Functions A, B, and C are now all methods of Class D.

### CODE REVIEW

File: mmeds/tests/server/test_server.py [hunk 1]
```diff
@@ -793,21 +793,25 @@ def test_fa_single_id(self):
         self.generate_aliquot_id()
 
     def generate_aliquot_id(self):
+        return
```
Comment (mstapylton) [inline mmeds/tests/server/test_server.py:796]: bare return at top of function?

File: mmeds/tests/unit/test_util.py [hunk 2]
```diff
@@ -17,6 +17,27 @@ class UtilTests(TestCase):
...
+                                                           subjects,
+                                                           'human')
+        for error in errors:
+            print(error)
+        for warning in warnings:
+            print(warning)
+        self.assertEqual(errors, [])
+        self.assertEqual(warnings, [])
+
+    @skip
```
Comment (mstapylton) [inline mmeds/tests/unit/test_util.py:40]: why are you now skipping these tests?

**Review by mstapylton — CHANGES_REQUESTED** [review]
Two questions

File: mmeds/tests/server/test_server.py (hunk 1 above)
Comment (DSWallach) [reply mmeds/tests/server/test_server.py:796]: It doesn't work, and I think the single ID aliquot generation functionality is gonna be dropped anyway so I don't want to spend time figuring out how to fix the test for it

File: mmeds/tests/unit/test_util.py (hunk 2 above)
Comment (DSWallach) [reply mmeds/tests/unit/test_util.py:40]: Oh i forgot to take those out

**Review by mstapylton — APPROVED** [review]

**Pull request merged**